wizard/WizardModeSelection.qml

```diff
@@ -119,6 +119,7 @@ Rectangle {
 
                 onMenuClicked: {
                     if(appWindow.persistentSettings.nettype == 0){
+                        appWindow.persistentSettings.pruneBlockchain = true;
```

Comment:
Switch from `Advanced mode` to `Simple mode (bootstrap)` and immediately switch back to `Advanced mode`. You have pruning enabled.

Reply:
got it